src/test/regress/expected/multi_unsupported_worker_operations.out

```diff
@@ -0,0 +1,372 @@
+
+\c - - - :worker_1_port
+SELECT stop_metadata_sync_to_node('localhost', :worker_2_port);
+ERROR:  operation is not allowed on this node
+HINT:  Connect to the schema node and run it again.
+\c - - - :master_port
+SELECT hasmetadata FROM pg_dist_node WHERE nodeport=:worker_2_port;
+ hasmetadata 
+-------------
+ t
+(1 row)
+
+SELECT stop_metadata_sync_to_node('localhost', :worker_2_port);
+ stop_metadata_sync_to_node 
+----------------------------
+ 
+(1 row)
+
+SELECT hasmetadata FROM pg_dist_node WHERE nodeport=:worker_2_port;
+ hasmetadata 
+-------------
+ f
+(1 row)
+
+\c - - - :worker_2_port
+DELETE FROM pg_dist_node;
+SELECT worker_drop_distributed_table(logicalrelid) FROM pg_dist_partition;
+ worker_drop_distributed_table 
+-------------------------------
+ 
+ 
+(2 rows)
+
+\c - - - :worker_1_port
+-- DROP TABLE
+DROP TABLE mx_table;
+ERROR:  operation is not allowed on this node
+HINT:  Connect to the schema node and run it again.
+CONTEXT:  SQL statement "SELECT master_drop_all_shards(v_obj.objid, v_obj.schema_name, v_obj.object_name)"
+PL/pgSQL function citus_drop_trigger() line 21 at PERFORM
+SELECT count(*) FROM mx_table;
+ count 
+-------
+     5
+(1 row)
+
+-- master_drop_distributed_table_metadata
+SELECT master_drop_distributed_table_metadata('mx_table'::regclass, 'public', 'mx_table');
+ERROR:  operation is not allowed on this node
+HINT:  Connect to the schema node and run it again.
+SELECT count(*) FROM mx_table;
+ count 
+-------
+     5
+(1 row)
+
+-- master_copy_shard_placement
+SELECT logicalrelid, shardid AS testshardid, nodename, nodeport 
+FROM pg_dist_shard NATURAL JOIN pg_dist_shard_placement
+WHERE logicalrelid = 'mx_table'::regclass AND nodeport=:worker_1_port
+ORDER BY shardid
+LIMIT 1 \gset
+INSERT INTO pg_dist_shard_placement (nodename, nodeport, shardid, shardstate, shardlength)
+VALUES ('localhost', :worker_2_port, :testshardid, 3, 0);
+SELECT master_copy_shard_placement(:testshardid, 'localhost', :worker_1_port, 'localhost', :worker_2_port);
+ERROR:  operation is not allowed on this node
+HINT:  Connect to the schema node and run it again.
+SELECT shardid, nodename, nodeport, shardstate 
+FROM pg_dist_shard_placement
+WHERE shardid = :testshardid
+ORDER BY nodeport;
+ shardid | nodename  | nodeport | shardstate 
+---------+-----------+----------+------------
+ 1270000 | localhost |    57637 |          1
+ 1270000 | localhost |    57638 |          3
+(2 rows)
+
+DELETE FROM pg_dist_shard_placement WHERE nodeport = :worker_2_port AND shardid = :testshardid;
+-- recover_prepared_transactions
+SELECT recover_prepared_transactions();
+ERROR:  operation is not allowed on this node
+HINT:  Connect to the schema node and run it again.
+-- master_get_new_placementid
+SELECT master_get_new_placementid();
+ERROR:  operation is not allowed on this node
+HINT:  Connect to the schema node and run it again.
+-- master_expire_table_cache
+SET citus.large_table_shard_count TO 8;
+SELECT master_expire_table_cache('mx_table');
```

Comment:
I have a tendency to disallow operations start with `master_` but in this case I think you are correct and in fact the purpose of the `master_expire_table_cache` to prevent wrong results after modifications so running it from a worker makes sense.